test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -5626,11 +5626,20 @@ BOOST_AUTO_TEST_CASE(uninitialized_internal_storage_function)
 			}
 		}
 	)";
-	compileAndRun(sourceCode, 0, "Test");
+	ALSO_VIA_YUL(
```

Comment:
Any chance we could move this into semantic tests? Would require two separate files, one with `compileViaYul: false` and the other with `true`.

Reply:
I would vote for moving them to `semanticTests` with two files.

Reply:
The problem is that the identifier returned by the old codegen will be flaky and change with optimizer changes. I can move the check into the code, though, if you like.

Reply:
I am fine with leaving it here in that case